### DISCUSSION TITLE
Uninstall Image Automation

### DISCUSSION THREAD

**Discussion by jwerre** [fluxcd/flux2#2708] [Q&A] [answered]
I did the bootstrap installation with the `--components-extra=image-reflector-controller,image-automation-controller`. What would be the best way to *uninstall* these?

**Comment by stefanprodan** [comment, marked as answer]
You can rerun bootstrap without those components, they will get deleted from Git, then Flux will remove them from the cluster. The minimum required components are **source-controller** and **kustomize-controller**, everything else can be removed. Note that you should delete from Git all the image automation custom resource you have created, do this before removing the controllers.

**Reply by jwerre** [reply]
Thanks for the quick response. I ran `bootstrap` without these components and, on second thought, I think I _do_ need them. 

Right now my flux deployment scans the image repositories and automatically updates the deployment versions to the latest image (based on the SemVer tag). I'd like to stop this behavior so I can update the image manually. To do this I just need to remove `ImageUpdateAutomation` resource and run bootstrap without the `image-automation-controller` is that correct?

**Reply by hiddeco** [reply]
You can simply suspend the resource using `.spec.suspend`, which will stop the automation process without removing the resource.
